<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Facebook – log in or sign up]]>
    </line>
  </reporter-output>
  <suite started-at="2025-01-30T13:04:39 IST" name="Default suite" finished-at="2025-01-30T13:04:45 IST" duration-ms="5705">
    <groups>
    </groups>
    <test started-at="2025-01-30T13:04:39 IST" name="Default test" finished-at="2025-01-30T13:04:45 IST" duration-ms="5705">
      <class name="script.DemoTest">
        <test-method is-config="true" signature="initReport()[pri:0, instance:script.DemoTest@3c9754d8]" started-at="2025-01-30T13:04:39 IST" name="initReport" finished-at="2025-01-30T13:04:39 IST" duration-ms="263" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initReport -->
        <test-method is-config="true" signature="preCondition(java.lang.reflect.Method,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:script.DemoTest@3c9754d8]" started-at="2025-01-30T13:04:39 IST" name="preCondition" finished-at="2025-01-30T13:04:43 IST" duration-ms="4122" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void script.DemoTest.testValidLogIn()]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[CHROME]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[NO]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://localhost:4444]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method signature="testValidLogIn()[pri:0, instance:script.DemoTest@3c9754d8]" started-at="2025-01-30T13:04:43 IST" name="testValidLogIn" finished-at="2025-01-30T13:04:44 IST" duration-ms="1158" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [adcb] but found [abc]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [adcb] but found [abc]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at script.DemoTest.testValidLogIn(DemoTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Facebook – log in or sign up]]>
            </line>
          </reporter-output>
        </test-method> <!-- testValidLogIn -->
        <test-method is-config="true" signature="postCondition(org.testng.ITestResult)[pri:0, instance:script.DemoTest@3c9754d8]" started-at="2025-01-30T13:04:44 IST" name="postCondition" finished-at="2025-01-30T13:04:45 IST" duration-ms="390" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testValidLogIn status=FAILURE method=DemoTest.testValidLogIn()[pri:0, instance:script.DemoTest@3c9754d8] output=Facebook – log in or sign up]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
        <test-method is-config="true" signature="publishReport()[pri:0, instance:script.DemoTest@3c9754d8]" started-at="2025-01-30T13:04:45 IST" name="publishReport" finished-at="2025-01-30T13:04:45 IST" duration-ms="298" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- publishReport -->
      </class> <!-- script.DemoTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
